tests/config.py

```diff
@@ -101,14 +101,22 @@
 ##############################
 # libvirt provider settings
 ##############################
+
 PROVIDER_LIBVIRT_CONNECTION = os.getenv(
     'ROOKCHECK_PROVIDER_LIBVIRT_CONNECTION', "qemu:///system")
-# the libvirt network that will be used
-PROVIDER_LIBVIRT_NETWORK = os.getenv(
-    'ROOKCHECK_PROVIDER_LIBVIRT_NETWORK', "default")
-# the qcow2 image that will be used for libvirt. the image must
-# contain cloud-init!
-PROVIDER_LIBVIRT_IMAGE = os.getenv('ROOKCHECK_PROVIDER_LIBVIRT_IMAGE', None)
+
+# A new bridge network will be created in libvirt with this range.
+PROVIDER_LIBVIRT_NETWORK_RANGE = os.getenv(
+    'ROOKCHECK_PROVIDER_LIBVIRT_NETWORK_RANGE', "192.168.124.0/24")
+
+# The qcow2 image that will be used for libvirt. The image must
+# contain cloud-init. This can be an system path or a URL which rookcheck will
+# download.
+PROVIDER_LIBVIRT_IMAGE = os.getenv(
+    'ROOKCHECK_PROVIDER_LIBVIRT_IMAGE',
+    "https://download.opensuse.org/distribution/leap/15.1/jeos/"
+    "openSUSE-Leap-15.1-JeOS.x86_64-15.1.0-OpenStack-Cloud-Current.qcow2"
```

Comment:
let's not use a hardcoded version here please...

Reply:
Use https://download.opensuse.org/distribution/leap/15.1/jeos/openSUSE-Leap-15.1-JeOS.x86_64-OpenStack-Cloud.qcow2

Reply:
This is in config.py, so it's not really hard coded. Improving the config is on the todo too. But yep, will update to the correct url.